tests/DbaaS/pages/dbaasActionsPage.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
     }
   },
 
-  async deleteXtraDBCluster(dbClusterName, k8sClusterName, deleteCompleted = true) {
+  async deleteXtraDBCluster(dbClusterName, k8sClusterName, isDeleted = true) {
```

Comment:
What is this boolean for? If false, then cluster will not be deleted? Do I understand it correctly?

Reply:
Removed, thanks.